### PR TITLE
#575 request req add lines

### PR DESCRIPTION
**Pull request by joshxg**
starts #575!

- This is adding a very basic modal opening for both request and response requisitions.
- I wanted to just add the modal as a self contained PR as it is really just boiler plate and not-interesting, so later PRs can have a better diff. I thought to also do response as that will be needed later also and this way I can just copy paste a bunch of code & the diff in general shouldn't be much harder with having both

### CODE REVIEW

**Comment by joshxg** [comment]
this will alsoclose #853

**Comment by joshxg** [comment]
Ah. This is actually merged from your auth PR, with main merged in also 🙄  

So.. the diff will be a lot friendlier once your auth PR is merged!

File: packages/requisitions/src/RequestRequisition/DetailView/ContentArea.tsx [hunk 1]
```diff
@@ -1,16 +1,24 @@
-import React, { FC } from 'react';
+import React from 'react';
```
Comment (joshxg) [inline packages/requisitions/src/RequestRequisition/DetailView/ContentArea.tsx:1]: Just adding the row click for the content area

File: packages/requisitions/src/RequestRequisition/DetailView/DetailView.tsx [hunk 2]
```diff
@@ -7,30 +7,57 @@ import {
   RouteBuilder,
   useNavigate,
   useTranslation,
+  useEditModal,
```
Comment (joshxg) [inline packages/requisitions/src/RequestRequisition/DetailView/DetailView.tsx:10]: This is the same as how the other areas open their modals

File: packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```
Comment (joshxg) [inline packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx:1]: just the building blocks of the modal

File: packages/requisitions/src/RequestRequisition/api/operations.generated.ts [hunk 4]
```diff
@@ -20,17 +20,17 @@ export type UpdateRequestRequisitionMutationVariables = Types.Exact<{
 
 export type UpdateRequestRequisitionMutation = { __typename: 'Mutations', updateRequestRequisition: { __typename: 'RequisitionNode', id: string } | { __typename: 'UpdateRequestRequisitionError' } };
 
-export type RequestRequisitionLineFragment = { __typename: 'RequisitionLineNode', id: string, itemId: string, requestedQuantity: number, calculatedQuantity: number, itemStats: { __typename: 'ItemStatsNode', averageMonthlyConsumption: number, stockOnHand: number, monthsOfStock: number } };
+export type RequestRequisitionLineFragment = { __typename: 'RequisitionLineNode', id: string, itemId: string, requestedQuantity: number, calculatedQuantity: number, itemStats: { __typename: 'ItemStatsNode', averageMonthlyConsumption: number, stockOnHand: number, monthsOfStock: number }, item: { __typename: 'ItemNode', id: string, name: string, code: string, unitName?: string | null } };
```
Comment (joshxg) [inline packages/requisitions/src/RequestRequisition/api/operations.generated.ts:23]: adding item fields to request req line

File: packages/requisitions/src/ResponseRequisition/DetailView/ContentArea.tsx [hunk 5]
```diff
@@ -1,16 +1,24 @@
-import React, { FC } from 'react';
+import React from 'react';
```
Comment (joshxg) [inline packages/requisitions/src/ResponseRequisition/DetailView/ContentArea.tsx:1]: then repeated all the same steps for a response req

**Review by mark-prins — APPROVED** [review]
👍  LGTM!

**Pull request merged**